src/cdk/a11y/aria-describer/aria-describer.ts

```diff
@@ -185,6 +185,13 @@ export class AriaDescriber implements OnDestroy {
 
       messagesContainer = this._document.createElement('div');
       messagesContainer.id = MESSAGES_CONTAINER_ID;
+      // We add `visibility: hidden` in order to prevent text in this container from
+      // being searchable by the browser's Ctrl + F functionality.
+      // Screen-readers will still read the description for elements with aria-describedby even
+      // when the description element is not visible.
+      messagesContainer.style.visibility = 'hidden';
+      // Even though we use `visibility: hidden`, we still apply `cdk-visually-hidden` so that
+      // the description element doesn't impact page layout.
```

Comment:
You should be able to remove the `messagesContainer.setAttribute('aria-hidden', canBeAriaHidden + '');` below since `visibility: hidden` hides the element from screen readers as well. I'm not sure whether the note about IE and Edge doesn't apply to `visibility` as well though.

Also the unit tests need to be updated to reflect the new changes.